test/integration_test/specs/kubevirt-fedora-multiple-disks-datavol-only/fedora-vm-multidisk-datavol-only.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  labels:
+    app: fedora-vm-multidisk-datavol-only
+    vm.kubevirt.io/template: fedora-server-small
+    vm.kubevirt.io/template.namespace: openshift
+  name: fedora-vm-multidisk-datavol-only
+spec:
+  running: true
+  template:
+    metadata:
+      annotations:
+        vm.kubevirt.io/flavor: small
+        vm.kubevirt.io/os: fedora
+        vm.kubevirt.io/workload: server
+      labels:
+        kubevirt.io/size: small
+    spec:
+      domain:
+        cpu:
+          cores: 1
+          sockets: 1
+          threads: 1
+        devices:
+          disks:
+          - bootOrder: 1
+            disk:
+              bus: virtio
+            name: rootdisk
+          - bootOrder: 2
+            disk:
+              bus: virtio
+            name: cloudinitdisk
+          networkInterfaceMultiqueue: true
+          rng: {}
+        features:
+          acpi: {}
+          smm:
+            enabled: true
+        firmware:
+          bootloader:
+            efi: {}
+        machine:
+          type: pc-q35-rhel9.2.0
+        resources:
+          requests:
+            memory: 2Gi
+      evictionStrategy: LiveMigrate
+      terminationGracePeriodSeconds: 180
+      volumes:
+      - dataVolume:
+          name: fedora-root-disk-datavol-only
+        name: rootdisk
+      - dataVolume:
+          name: fedora-data-disk-datavol-only
+        name: fedora-datavol-disk
+      - cloudInitNoCloud:
+          userData: |-
+            #cloud-config
+            user: fedora
+            password: password1
+            chpasswd: { expire: False }
+        name: cloudinitdisk
+  dataVolumeTemplates:
+  - metadata:
+      name: fedora-root-disk-datavol-only
+    spec:
+      source:
+        pvc:
+          name: fedora-template-pvc
+          namespace: openshift-virtualization-os-images
```

Comment:
Can we add another datavolume with "blank" source?
```
blank: {}
```

Reply:
Fixed.